packages/assets-controllers/src/multi-chain-accounts-service/types.ts

```diff
@@ -0,0 +1,27 @@
+export type GetBalancesQueryParams = {
+  /** @description Comma-separated network/chain IDs */
+  networks?: string;
+  /** @description Whether or not to filter the assets to contain only the tokens existing in the Token API */
+  filterSupportedTokens?: boolean;
+  /** @description Specific token addresses to fetch balances for across specified network(s) */
+  includeTokenAddresses?: string;
+  /** @description Whether to include balances of the account's staked asset balances */
+  includeStakedAssets?: boolean;
+};
+
+export type GetBalancesResponse = {
+  count: number;
+  balances: {
+    object: string;
+    address: string;
+    symbol: string;
+    name: string;
+    type?: string;
+    timestamp?: string;
+    decimals: number;
+    chainId: number;
+    balance: string;
```

Comment:
This was retrieved from testing the API.
I'll add some JSDoc or expand this once we start using this data.